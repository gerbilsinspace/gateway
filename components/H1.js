import styled from 'styled-components';

const H1 = styled.h1`
  color: #fff;
  font-size: 40px;
`;

export default H1;
