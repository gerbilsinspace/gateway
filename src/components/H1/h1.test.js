import React from 'react';
import { shallow } from 'enzyme';
import H1 from './index';

let component;

describe('H1', () => {
  beforeEach(() => {
    component = shallow(<H1 />);
  });

  it('should render without crashing', () => {
    expect(component).toBeDefined();
  });

  it('should match snapshot with no data', () => {
    expect(component).toMatchSnapshot();
  });
})