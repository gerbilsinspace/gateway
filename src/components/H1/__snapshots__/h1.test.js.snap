// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H1 should match snapshot with no data 1`] = `
<h1
  className="sc-bdVaJa bDWFJH"
/>
`;
